server/plugin.go

```diff
@@ -138,14 +169,12 @@ func (p *Plugin) verifyJenkinsCredentials(username, token string) (bool, error)
 // createEphemeralPost creates an ephemeral post
 func (p *Plugin) createEphemeralPost(userID, channelID, message string) {
 	post := &model.Post{
-		UserId:    userID,
+		UserId:    p.botUserID,
 		ChannelId: channelID,
 		Message:   message,
 		Type:      model.POST_DEFAULT,
 		Props: map[string]interface{}{
-			"from_webhook":      "true",
-			"override_username": jenkinsUsername,
-			"override_icon_url": p.getConfiguration().ProfileImageURL,
+			"from_webhook": "true",
```

Comment:
Why is this Prop needed? It way only needed because of `override_username`, right?